### ISSUE TITLE
Atlas product animation — NFT viewer side

### ISSUE THREAD

**Issue by toiletgranny** [Joystream/atlas#2287]
The goal of this task is to create a short product animation for social media to advertise NFT functionalities being added to Atlas, focusing on the viewer experience.

1. Animation length: `30-60s`,
2. Animation resolution: `1080 x 1080`,
3. Sound: `no`,
4. Animation style: https://github.com/Joystream/atlas/issues/1844#issuecomment-1026853545
5. Content on mockups: `100% fake`,
6. Example scope of the animation contents:
   - [Channel view](https://www.figma.com/file/POw4j95jszfIP5bGfas2IC/Channel-page) with a particular focus on NFT tiles, 
   - [Video page](https://www.figma.com/file/sMydmZJlzhK91FTIPOd7D2/Video-page) with a particular focus on the NFT widget,
   - [NFT purchase view](https://www.figma.com/file/wjeAOZq77pSUFZsYXJqKIB/NFT-purchase-view),
   - [Notifications](https://www.figma.com/file/Goequb9eZDL6WxK7T74oxS/Notifications), with a particular focus on NFT alerts.

See also: https://github.com/Joystream/atlas/issues/2288

**Comment by toiletgranny** [comment]
_👇 🔉  sound on_

https://user-images.githubusercontent.com/35307309/159439214-4a7132ea-03fc-4850-a367-1d43d9acdd20.mp4


@bedeho @mrexpat the animation is ready for you guys to be reviewed. Since it's the first time we're doing this together, below are things you need to know:

1. **Text:** All text screens in the animation can be considered placeholders for your final, revised copy. The headlines I'm providing in the video aim to be a loose suggestion for you guys on the length of the copy and also an overarching theme of each of the text screens.
2. **Content:** All the contents in the mockups are 100% fake and created for the purpose of this video, with a goal of being reused in future product animations and potentially Figma designs going forward.
   - The channel and videos featured in the animation are from a fake indie rock band called _The Joystreamers_. All song titles and names of the bands which are featured in their songs have been completely made up so that there is no resemblance to any real-world's copyrighted content. The mockups from the animation [can be inspected in this Figma file](https://www.figma.com/file/D3wcgz6GFw0MPWjxznD5BK/NFT-(Viewer)?node-id=0%3A1).
   - All videos and photographs used in the animation come from [Unsplash](https://unsplash.com/license) and [Pexels](https://www.pexels.com/license/) and are free to use.
   - Of course, all of that can be changed if you have a clear and specific idea on what other channels and videos we could be showing instead. To update the mockups and the animation with completely new content would take around **4-6 hours**. Updating singular videos, or changing the channel's name/cover/avatar would take around an hour to complete. 
4. **Usernames and avatars:** Regarding usernames and avatars featured in the animation, it all comes from the list of Founding Members with the addition of Bedeho, Mokhtar, and Martin. This can be changed to something else and will take around **2 hours** to complete. 
5. **Sound:** @bedeho I know we agreed to not have any sound or music in the animations, but... well, I just couldn't stop myself. It's just so much easier to find a tempo and rhythm when you're editing with an audio track in the background.
   - Of course, [there's a muted version that can be found here](https://github.com/Joystream/design/blob/master/social-media/product-animations/atlas/1_nft_viewer/_render/1_nft_viewer_muted.mp4) and it's the one I'm sure we'll end up using.
   - However, if you guys end up liking the version with an audio track, then the good news is: the licensing is already covered by Netguru, thanks to having an Artlist.io subscription. 
6. **Stills and GIFs:** Upon request, I can export a set of still frames and/or short GIFs from the animation to be reused in social media. This should take around an hour. Let me know if you want that and if there are any specific moments in the animation you'd really like to see as stills or GIFs.
7. **Resolution and aspect ratio:** I ended up choosing a resolution of `1440x1080`, which translates to a `4:3` aspect ratio, as this turned out to be a much better use of the real-estate considering that the mockups for showcasing the designs are also predominantly horizontal. Hope that's not a problem.
   - Upon request, I can create an additional `1:1` version if that's needed, or any other aspect ratio version (`16:9`, `16:10`, `2.39:1`). This should not take more than **15 minutes** (per version) thanks to the way we built our animation templates (lots of flexibility and fast, effortless customizability).

**Comment by toiletgranny** [comment]
@MattAugustynowicz since we've been working together on #1844, your input on the above would be also very much appreciated. Let me know if there's anything you'd suggest reconsidering from the technical/visual standpoint. 🙏

**Comment by bedeho** [comment]
This was amazing, so excited to announce this!, the audio track actually really made it pop! Why did we decide against it?

The way you animated the UI to show unique features, in a way that was not just the standard behaviour of the app, I think it was transformative for the demo, it really guides the users attention to the important parts of an otherwise complex and novel scene.  This style just worked really well.

**Comment by MattAugustynowicz** [comment]
@toiletgranny I won't look at your stuff again, or I'll fall into huge complexes 😄 
Looks perfect, I absolutely love it! Awesome job sir 🙇

**Comment by toiletgranny** [comment]
Thank you! I'm so glad to know you liked it!

> This was amazing, so excited to announce this!, the audio track actually really made it pop! Why did we decide against it?

The key reason was the time to produce such an animation; we were hoping that a video with no sound takes less time to produce. In general, this still might be true. Still, in this case, when we're working on a relatively short and very dynamic animation, I think the presence of an audio track speeds up the process, as it takes away some decision making on the aspects of timing the animation (instead of working with silence, you let the audio track guide your pace, movement, transitions, etc.). 

Anyway, so it seems like we have a decision on the audio. Fantastic! 

Any suggestions in regards to the copy layer of the video? Perhaps this is where @mrexpat would be willing to kick in with some recommendations.

**Comment by mrexpat** [comment]
First off: love the professionalism. Great job. 

Some specifics:

00:00 - 00:05 these are the most important seconds in a video like this. This is where the viewer is trying to figure out "who is this for? and why should I give a shit?" Therefor, IMO this video should start with the "introducing video NFTs" frame, because that answers both of those questions

00:06+ this is where you currently show the video NFTs and tab and who is collecting the NFTs, but I don't think these are really selling points or interesting points to make.. what comes after that is more interesting, and ideally what we would show here is the selling/issuing process and the buying process of NFTs.. and showing the creator making money.. what people care about here is 1) how can I make money from this, so they want to see this in action (creators) and 2) how buyers can make money from it by buying and reselling the NFTs..

So this comes a bit later around 00:23 - I think this should be earlier like first 10 seconds

TL;DR - IMO this video should start with the "introducing video NFTs frame" then basically show the 00:22 - 00:47 footage. The 00:00 - 00:23 footage could be dropped entirely without losing much effect, or it could be kept, but in the latter half of the video instead of the first half.

My 2 cents!

**Comment by toiletgranny** [comment]
Thanks for your feedback, Robert, I appreciate it a lot! 

Firstly, I should have probably started by letting you know that this is the first out of two animations we have planned for NFTs on Atlas. While this video focuses on the experience from the collectors/viewers' perspective, it's the second one that will be focusing more on NFTs from the perspective of creators.

> 00:00 - 00:05 these are the most important seconds in a video like this

Right, of course, it is a common pattern to try to squeeze in the core message within the first 5 seconds of a video when it's a YouTube ad or any other video ad which can be skipped/dismissed within the first X seconds. 

With this video, I guess I didn't really consider it might fall under the above-mentioned criteria, as — to my understanding — having a goal of creating a product animation means more of trying to attract and keep the viewer's attention through intriguing visuals, a compelling story, and gradually revealing more puzzles, rather than attempting to _sell_ as much as possible as quickly as possible.

That's why, in the original version, the story is being unpacked in a very specific order, reflecting a typical user's journey:
1. You first discover NFTs on a channel page,
2. You then learn more about a specific NFT from its video page,
3. From the video page you initiate a transaction to either buy or bid for an NFT,
4. As an NFT owner, you receive a notification about bids on your NFTs,
5. And finally you go back to the video page to settle your action and gain profits.

---

Having said that, I did prepare an alternative cut, based on your feedback, for you to see if you indeed like it more or think it will get more attention. I had to edit the track in the background a bit, tweak the motion here and there, but I hope it looks and sounds okay! Looking forward to seeing your thought on this. 🚀

https://user-images.githubusercontent.com/35307309/160167106-e1713a49-8cee-45a1-a7a3-567555b8a3b0.mp4

**Comment by mrexpat** [comment]
This does a much better job at giving the viewer what they want imo.. the part from 00:01 to 00:12 could be removed imo..

**Comment by toiletgranny** [comment]
Great! Robert, while I'm somewhat concerned about stripping down the video and not devoting enough time to set the viewer in context, I fully trust your judgment on this and I acknowledge there might be scenarios in which a more condensed cut is better suited.

Below you'll find two versions that I believe can be used interchangeably, depending on the context:

1. `20s` A version that focuses solely on the process of purchasing an NFT, just as you requested. I also dropped all scenes with the channel page based on your initial suggestion. Super short, straight to the point!

https://user-images.githubusercontent.com/35307309/161089866-ef68b071-9c2e-4cba-a379-54a52621fe7d.mp4

2. `45s` A more relaxed version takes its time to introduce the viewer to each scene and does it in an order that makes sense. We can consider it a "director's cut"! 😅 I did, however, apply minor adjustments here and there, partially based on your previous feedback, eg. the video begins with words the "Video NFTs" (instead of "Introducing), so that it's instantly clear what is the viewer about to see.  Best suited wherever time is not a burning issue, eg. Discord, regular Twitter posts, keynotes, etc. 

https://user-images.githubusercontent.com/35307309/161090625-571f70a9-14e5-4423-ae72-73f358f7dc10.mp4

---

Please, let me know if there's anything you'd like to change about the copy. Also, keep in mind that all animations can be found in the `design` repo on GitHub, [in this folder.](https://github.com/Joystream/design/tree/master/social-media/product-animations/atlas/1_nft_viewer/_render)

@mrexpat

**Comment by mrexpat** [comment]
Man, this is fantastic.. for me the first option is a no brainer winner.. short, punchy, keeps engagement throughout.. this one is a home run

For me the 2nd option won't do as good a job of capturing attention and getting the punchline across..

Really great job on this!

**Comment by toiletgranny** [comment]
Awesome! 🔥  Glad you liked it! I would be still hoping to see the 2nd version used under the right circumstances. ❤️

If there are no other comments or requests for now, and the copy is fine with everyone then, @dmtrjsg, I believe this issue can be now closed.
